src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/global.yml

```diff
@@ -317,3 +317,41 @@ Resources:
               - "*"
       Roles:
         - !Ref ReadOnlyAutomationRole
+  ADFTerraformRole:
```

Comment:
Is this the role that the deployment pipeline assumes into to apply its changes?
If so, the comments do not reflect that at the moment. As it is talking about being deployed in the deployment account specifically.

Could you change this to be stored inside the example global iam file, with a clear instruction in the documentation how they could enable that by copying it to the global-iam.yml file?